### DISCUSSION TITLE
Is it possible to show the full options list when the autoCompleteJs.input has focus?

### DISCUSSION THREAD

**Discussion by NeilSchneider** [TarekRaafat/autoComplete.js#277] [General]
Please ignore, found the threshold property.
I would like the users to see all items available if when the search input is focused and no input has been entered.  Is this possible.
Thanks

**Comment by TarekRaafat** [comment]
Hello @NeilSchneider,

Try using the below snippet, and let me know how it goes.

```js
events: {
   input: {
      focus() => autoCompleteJS.start(" ");
  },
},
```

**Reply by radishfm** [reply]
Add these in documentation, please. I think this is a frequent case.

**Reply by TarekRaafat** [reply]
Hello @radishfm,

It is in the documentation under `How-to Guides` -> [Open List on Focus](https://tarekraafat.github.io/autoComplete.js/#/how-to-guides?id=open-list-on-focus)

**Reply by Joanzo** [reply]
This is not working if there's no space in the list.
i.e: [
"Apple", --> this will NOT be shown,
"Banana Orange" --> will be shown
]

**Comment by LucasRoquilly** [comment]
Hello, I added a solution here : https://github.com/TarekRaafat/autoComplete.js/issues/332#issuecomment-1141177782

Working codepen : https://codepen.io/lucasroquilly/pen/LYQdpXB
